vendor.mod

```diff
@@ -165,15 +165,8 @@ require (
 	google.golang.org/protobuf v1.28.0 // indirect
 )
 
-replace (
-	// More recent versions result in a panic in libnetwork.
-	// FIXME(thaJeztah): we need to fix how we use this library or replace it; see https://github.com/moby/moby/issues/43753
-	github.com/armon/go-radix => github.com/armon/go-radix v0.0.0-20150105235045-e39d623f12e8
-	// Resolve dependency hell with github.com/cloudflare/cfssl (transitive via
-	// swarmkit) by pinning the certificate-transparency-go version. Remove once
-	// module go.etcd.io/etcd/server/v3 has upgraded its dependency on
-	// go.opentelemetry.io/otel to v1.
-	github.com/google/certificate-transparency-go => github.com/google/certificate-transparency-go v1.0.20
-	// Removes etcd dependency
-	github.com/rexray/gocsi => github.com/dperny/gocsi v1.2.3-pre
-)
+// Resolve dependency hell with github.com/cloudflare/cfssl (transitive via
+// swarmkit) by pinning the certificate-transparency-go version. Remove once
+// module go.etcd.io/etcd/server/v3 has upgraded its dependency on
+// go.opentelemetry.io/otel to v1.
+replace github.com/google/certificate-transparency-go => github.com/google/certificate-transparency-go v1.0.20
```

Comment:
You don't need this anymore, etcd upgraded to otel v1.

Reply:
@thaJeztah https://github.com/etcd-io/etcd/blob/server/v3.5.5/server/go.mod#L35

Reply:
Yes, working on that one (currently struggling with SwarmKit CI), so I'll do that in a follow-up 😄